tests/classes/PluginManager.js

```diff
@@ -765,8 +766,11 @@ describe('PluginManager', () => {
   describe('Plugin/CLI integration', () => {
```

Comment:
oh also, does it make sense to move this test to the integration_test.js file instead of here?